integration-tests/src/tests/client/features/wallet_contract.rs

```diff
@@ -219,7 +219,8 @@ fn test_transaction_from_eth_implicit_account_fail() {
     assert_eq!(response, expected_tx_error);
 
     // Try to deploy the Wallet Contract again to the ETH-implicit account. Should fail because there is no access key.
-    let wallet_contract_code = wallet_contract(chain_id, PROTOCOL_VERSION).code().to_vec();
+    let magic_bytes = wallet_contract_magic_bytes(&chain_id, PROTOCOL_VERSION);
+    let wallet_contract_code = wallet_contract(*magic_bytes.hash()).unwrap().code().to_vec();
```

Comment:
There's an `unwrap()` that will cause other chains to fail, would be nicer to return an error.